www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf

```diff
@@ -19,7 +19,7 @@ virus_scan.Allow204Responces on
 {% if helpers.exists('OPNsense.cicap.antivirus.passonerror') and OPNsense.cicap.antivirus.passonerror != '' %}
 virus_scan.PassOnError off
 {% endif %}
-{% if helpers.exists('OPNsense.cicap.antivirus.cmaxobjectsize') and OPNsense.cicap.antivirus.maxobjectsize != '' %}
+{% if helpers.exists('OPNsense.cicap.antivirus.maxobjectsize') and OPNsense.cicap.antivirus.maxobjectsize != '' %}
 virus_scan.MaxObjectSize  5M
```

Comment:
I could be wrong (and sorry for hijacking this PR), but shouldn't this line be something like 

```
virus_scan.MaxObjectSize {{ OPNsense.cicap.antivirus.maxobjectsize }}
```

similar to line 7? If yes, all the other options have to be fixed as well. At the moment everything seems to be hardcoded.

Reply:
yes, you are right:

https://github.com/opnsense/plugins/blob/master/www/c-icap/src/opnsense/mvc/app/models/OPNsense/CICAP/Antivirus.xml#L40-L43